spec/data-model/README.md

```diff
@@ -1,15 +1,33 @@
 # DRAFT MessageFormat 2.0 Data Model
 
-To work with messages defined in other syntaxes than that of MessageFormat 2,
-an equivalent data model representation is also defined.
-Implementations MAY provide interfaces which allow
-for MessageFormat 2 syntax to be parsed into this representation,
-for this representation to be serialized into MessageFormat 2 syntax
-or any other syntax,
-for messages presented in this representation to be formatted,
-or for other operations to be performed on or with messages in this representation.
+This section defines a data model representation of MessageFormat 2 _messages_.
 
 Implementations are not required to use this data model for their internal representation of messages.
+Neither are they required to provide an interface that accepts or produces
+representations of this data model.
+
+The reason this specification provides a data model is to allow interchange of 
+the logical representation of a _message_ between different implementations.
+This includes mapping legacy formatting syntaxes (such as MessageFormat 1)
+to a MessageFormat 2 implementation.
+Another use would be in converting to or from translation formats without 
+the need to continually parse and serialize all or part of a message.
+
+Implementations that expose APIs supporting the production or consumption of a
+_message_ as a data structure are encouraged to use this data model.
+
+This data model provides these capabilities:
+- any MessageFormat 2 message can be parsed into this representation
+- any well-formed data model representation can be serialized as a well-formed
```

Comment:
I see that well-formedness is defined for messages in [syntax.md](https://github.com/unicode-org/message-format-wg/blob/60ab652d11a2f15741a76853f21f552e758556d3/spec/syntax.md#well-formed-vs-valid-messages), but is well-formedness defined for data models? I don't see where.

Reply:
It's not. If we do define well-formedness, it'd be best to do so somehow via the syntax deifintions, because otherwise there's potential for missing out e.g. how a reserved-statement body can't contain an `@` character.

Reply:
Fixed in a separate commit